jupyter-notebooks/orders_basemaps/requests_order_basemaps.ipynb

```diff
@@ -0,0 +1,605 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "response = requests.get(orders_url, auth=session.auth)\n",
+    "print(response)\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Create an order packet\n",
+    "---\n",
+    "Package up the details of your order in a JSON object and make a POST request, passing in the Orders URL, your JSON, your API key, and the content-type header. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "order_params = {\n",
+    "    \"name\": \"basemap order with geometry\",\n",
+    "    \"source_type\": \"basemaps\",\n",
+    "    \"order_type\": \"partial\",\n",
+    "    \"products\": [\n",
+    "        {\n",
+    "            \"mosaic_name\": \"global_monthly_2022_01_mosaic\",\n",
+    "            \"geometry\": {\n",
+    "                \"type\": \"Polygon\",\n",
+    "                \"coordinates\": [\n",
+    "                   [\n",
+    "                       [4.607406, 52.353994],\n",
+    "                       [4.680005, 52.353994],\n",
+    "                       [4.680005, 52.395523],\n",
+    "                       [4.607406, 52.395523],\n",
+    "                       [4.607406, 52.353994]\n",
+    "                   ]\n",
+    "                ]\n",
+    "            }\n",
+    "        }\n",
+    "    ],\n",
+    "    \"tools\": [\n",
+    "        {\"merge\": {}},\n",
+    "        {\"clip\": {}}\n",
+    "\n",
+    "\n",
+    "    ],\n",
+    "    \"delivery\": {\n",
+    "        \"google_cloud_storage\": {\n",
+    "            \"bucket\": \"devrel-notebooks\",\n",
+    "            \"credentials\": GCP_CREDENTIALS,\n",
+    "            \"path_prefix\": \"basemaps-to-cloud/\"\n",
+    "        }\n",
+    "\n",
+    "\n",
+    "    }\n",
+    "}\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Print the response object\n",
+    "---\n",
+    "Retrieve the response object as text to print out if there are no errors."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ORDERS_API_URL = 'https://api.planet.com/compute/ops/orders/v2'\n",
+    "headers = {'content-type': 'application/json'}\n",
+    "param_req = requests.post(ORDERS_API_URL, data=json.dumps(\n",
+    "    order_params), auth=auth, headers={'content-type': 'application/json'})\n",
+    "\n",
+    "print(param_req.text)\n"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Place the order\n",
+    "---\n",
+    "The following code places the order."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def place_order(order_params, auth):\n",
```

Comment:
reformat this function to look like
```python
def place_order(order_params, auth):
    response = requests.post(ORDERS_API_URL,
                             data=json.dumps(order_params),
                             auth=auth,
                             headers=headers)
    order_id = response.json()['id']
    order_url = ORDERS_API_URL + '/' + order_id
    return order_url
```

Reply:
Also, would be helpful to do a little docstring which describes the function, its arguments, and its output. Something with this format is typical:
```python
    '''Description of the function.

    Parameters:
        param_1: Description of the param_1.
        param_2: Description of the param_2.
        
    Returns:
        return_item: Description of return_item,
    '''
```

Reply:
Putting this together could looks something like this:

```python
def place_order(order_params, auth):
    '''Sends a POST request to the Orders API.

    Parameters:
        order_params: Details of the order.
        auth: Basic authentication object.
        
    Returns:
        order_url: The order's URL.
    '''
    response = requests.post(ORDERS_API_URL,
                             data=json.dumps(order_params),
                             auth=auth,
                             headers=headers)
    order_id = response.json()['id']
    order_url = ORDERS_API_URL + '/' + order_id
    return order_url
```